org.lflang/src/org/lflang/generator/c/CGenerator.xtend

```diff
@@ -1158,11 +1158,8 @@ class CGenerator extends GeneratorBase {
     }
     
     /**
-     * Add the platform files to 'coreFiles'.
-     * 
-     * Also if useCmake is set to false, detect the host OS
-     * and add the appropriate platform support file(s) to 
-     * the compile command (compileAdditionalSources).
+     * Add the appropriate platform files to 'coreFiles'. These platform files
```

Comment:
Thanks for fixing this daunting merge conflict. I think the old comment reflects the intent of this function more accurately.